pvr.hts/resources/settings.xml

```diff
@@ -14,6 +14,11 @@
 	<setting id="epg_async" type="bool" label="30101" default="false"/>
   </category>
 
+  <category label="30300">
+	<setting id="pretuners" type="number" label="30301" default="1" />
+	<setting id="pretuner_closedelay" type="number" label="30302" default="10" />
```

Comment:
Both of these need `option="int"` as well, see https://github.com/Jalle19/pvr.hts/commit/4aad8b6aa15d9c30d5eced22f5fa43c19b16f7ee